app/src/shared/components/TextField/NumberField/index.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React, { useCallback, useState } from 'react'
+import cx from 'classnames'
+
+import SvgIcon from '$shared/components/SvgIcon'
+
+import styles from '../textField.pcss'
+
+type Props = {
+    className?: string,
+    type?: string,
+    min?: number,
+    max?: number,
+    step?: number,
+    hideButtons?: boolean,
+}
+
+const NumberField = ({ className, hideButtons, ...props }: Props) => {
+    const [value, setValue] = useState(null)
```

Comment:
@tumppi, how do you pass a value to the component? For instance, what's gonna happen if I do this `<NumberField value={42} />`? Will it show `42` in the field?